tests/Makefile

```
@@ -53,24 +53,18 @@ TEST_HAPPY_OPTS = --strict
 %.n.hs : %.ly
 	$(HAPPY) $(TEST_HAPPY_OPTS) $< -o $@
 
-%.g.hs : %.ly
-	$(HAPPY) $(TEST_HAPPY_OPTS) -g $< -o $@
-
-%.gc.hs : %.ly
-	$(HAPPY) $(TEST_HAPPY_OPTS) -gc $< -o $@
+%.c.hs : %.ly
+	$(HAPPY) $(TEST_HAPPY_OPTS) -c $< -o $@
 
 %.n.hs : %.y
 	$(HAPPY) $(TEST_HAPPY_OPTS) $< -o $@
```

Note that this rule as well as for `%.c.hs` appear to be redundant. Perhaps remove.

---

Done.

---

You removed *both* rules for `%.c.hs` (only one of which was redundant), but left *both* rules for `%.n.hs` (one of which is still redundant).
I suggest the following change:

```suggestion
%.c.hs : %.y
	$(HAPPY) $(TEST_HAPPY_OPTS) -c $< -o $@
```

---

Nevermind, I *totally* missed that we need two rules: One to build \*.y files, the other to build \*.ly files.

I reinstated the rules for \*.c.hs and will merge ASAP.